pkg/cmd/controller/pipelinerunner.go

```diff
@@ -59,6 +59,8 @@ type PipelineRunnerOptions struct {
 	UseMetaPipeline      bool
 	MetaPipelineImage    string
 	SemanticRelease      bool
+	noApply              bool
```

Comment:
For the pipelinerunner these options really don't make much sense. They are relevant for the transitive command which is called under the hood. Hence, the env variable approach is better.